File: Java/Card.java
class Card extends Payment {
    
    String cardNumber;
    String cardHolder;
    String dateExp;
    String ccv;

    public Card(String cardNumber, String cardHolder, String dateExp, String ccv){
        this.cardNumber = cardNumber;
        this.cardHolder = cardHolder;
        this.dateExp = dateExp;
        this.ccv = ccv;
    }
}

File: PHP/card.php
<?php
class Card extends Payment{
    public $cardNumber;
    public $cardHolder;
    public $dateExp;
    public $ccv;

    public function __construct($cardNumber, $cardHolder, $dateExp, $ccv)
    {
        $this->cardNumber = $cardNumber;
        $this->cardHolder = $cardHolder;
        $this->dateExp = $dateExp;
        $this->ccv = $ccv;
    }
}
